.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Could you please exclude this file and the tasks.json? Each kata has their own, and it can be confusing to add such files to the root folder, since they are configured to call BasicGates kata specifically.

Reply:
ok, i think i got rid of lanch.json and tasks.json from my branch.  i will go through the rest of the Kata and make the other improvements